frontend/src/components/Divisions.tsx

```diff
@@ -95,26 +101,11 @@ const Divisions = ({
   onRefetch,
 }: DivisionsProps) => {
   const [isAddModalOpen, setIsAddModalOpen] = useState<boolean>(false);
-  const [isEditModalOpen, setIsEditModalOpen] = useState<boolean>(false);
+  const [selectedIdForEdit, setSelectedIdForEdit] = useState<string>();
 
-  // component actions
-  const onAdd = () => {
-    setIsAddModalOpen(true);
-  };
-
-  // row actions
-  const onView = (row: IDivision) => {
-    alert(`Not yet implemented, ${row}`);
-  };
-
-  const onEdit = (row: IDivision) => {
-    console.log(`Not yet implemented, ${row}`);
-    setIsEditModalOpen(true);
-  };
-
-  const onDelete = (row: IDivision) => {
-    alert(`Not yet implemented, ${row}`);
-  };
+  const { mutateAsync: addDivisionMutation } = useAddDivisionMutation();
```

Comment:
We need to handle loading state, while the request is done the table should be showing the loading icon.